source/reference/mongosqld.txt

```diff
@@ -1023,25 +1031,27 @@ with each.
      - ``mongosqld`` Options
      - Behavior
 
-   * - Standalone Reader
-     - - :option:`--sampleMode <mongosqld --sampleMode>` ``read``
-       - :option:`--sampleSource <mongosqld --sampleSource>` *not used*
+   * - Standalone Schema
+     - - :option:`--schemaMode <mongosqld --schemaMode>` ``custom``
+       - :option:`--schemaSource <mongosqld --schemaSource>` *not used*
      - :manual:`Standalone </reference/glossary/#term-standalone>`
        :program:`mongod` instance. :binary:`~bin.mongosqld`
-       samples data upon startup. If :option:`--sampleRefreshIntervalSecs
-       <mongosqld --sampleRefreshIntervalSecs>` > 0, resample
+       samples data upon startup. If :option:`--schemaRefreshIntervalSecs
+       <mongosqld --schemaRefreshIntervalSecs>` > 0, resample
        at specified interval.
 
        .. example::
 
           .. code-block:: sh
 
-             mongosqld --sampleMode read
+             mongosqld --schemaMode custom
+
+       For more information, see :ref:`bi-cached-sampling`.
 
-   * - Clustered Reader
-     - - :option:`--sampleMode <mongosqld --sampleMode>` ``read``
-       - :option:`--sampleSource <mongosqld --sampleSource>` <dbName>
-       - :option:`--sampleRefreshIntervalSecs <mongosqld --sampleRefreshIntervalSecs>` *not used*
+   * - Custom Schema
+     - - :option:`--schemaMode <mongosqld --schemaMode>` ``custom``
+       - :option:`--schemaSource <mongosqld --schemaSource>` <dbName>
+       - :option:`--schemaRefreshIntervalSecs <mongosqld --schemaRefreshIntervalSecs>` *not used*
      - In a :manual:`replica set </replication/>` or
        :manual:`sharded cluster </sharding>`
        environment, sample data only at startup. Read schema data
```

Comment:
Custom mode doesn't sample at all, but rather reads a stored schema from the `schemaSource` db